src/solver/types.jl

```diff
@@ -325,6 +325,7 @@ end
 function AtmosConfig(
     s = argparse_settings();
     parsed_args = parse_commandline(s),
+    comms_ctx = get_comms_context(parsed_args),
```

Comment:
Hey @charleskawczynski ! Thanks for the prompt fix! Did you see #1790 ? We probably both opened a PR in the same exact moment! :laughing: 

@LenkaNovak and I pushed that fix in #1790 together. If you guys think that this one is better and supersedes #1790, then please go ahead and close that one. I am going to be off starting shortly. Thank you.

Reply:
Okay, I closed #1789  as superseded. Thank you both.

Reply:
Oh, I didn't see it, sorry! Either one is fine!